contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -520,7 +519,7 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
 
       //if stale update then revert
```

Comment:
stale comment, please update with the reasoning why we skip stale prices

Reply:
https://github.com/smartcontractkit/chainlink/pull/14981/commits/ded0b296ced82eb3f14a1e340677644dbf52cf47#diff-1859122b5ff5261692222f5ba921ee9108438593b0664e5aba8d573d866f27aaL520-R521